<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>customerbooksaintDialog</class>
 <widget class="QDialog" name="customerbooksaintDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgb(0, 255, 0)</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_identitysaint">
   <property name="geometry">
    <rect>
     <x>154</x>
     <y>170</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;MS Sans Serif&quot;;
background:rgb(85, 85, 127)</string>
   </property>
   <property name="placeholderText">
    <string>   book or booked</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_gmailsaint">
   <property name="geometry">
    <rect>
     <x>74</x>
     <y>50</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;Bell MT&quot;;
background:rgb(85, 85, 127)</string>
   </property>
   <property name="text">
    <string>   gmail</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_gmailsaint">
   <property name="geometry">
    <rect>
     <x>154</x>
     <y>50</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;Bell MT&quot;;
background:rgb(85, 85, 127)</string>
   </property>
   <property name="placeholderText">
    <string>  Joey  @gmail.com</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_identitysaint">
   <property name="geometry">
    <rect>
     <x>74</x>
     <y>174</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background:rgb(85, 85, 127)</string>
   </property>
   <property name="text">
    <string>  identity</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_eidsaint">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;Bell MT&quot;;
background:rgb(85, 85, 127)</string>
   </property>
   <property name="text">
    <string>   eid</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_eidsaint">
   <property name="geometry">
    <rect>
     <x>156</x>
     <y>110</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;Bell MT&quot;;
background:rgb(85, 85, 127)</string>
   </property>
   <property name="placeholderText">
    <string>  ctg01</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_updatecustomersaint">
   <property name="geometry">
    <rect>
     <x>188</x>
     <y>240</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;MS Shell Dlg 2&quot;;
background:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>book</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
